src/txn/txn.c

```diff
@@ -1822,7 +1841,8 @@ __wt_verbose_dump_txn(WT_SESSION_IMPL *session)
         WT_RET(__wt_msg(session,
           "ID: %" PRIu64 ", pinned ID: %" PRIu64 ", metadata pinned ID: %" PRIu64 ", name: %s", id,
           s->pinned_id, s->metadata_pinned, sess->name == NULL ? "EMPTY" : sess->name));
-        WT_RET(__wt_verbose_dump_txn_one(session, &sess->txn, 0, NULL));
+        WT_RET(
+          __wt_verbose_dump_txn_one(session, &sess->txn, WT_SESSION_TXN_SHARED(sess), 0, NULL));
```

Comment:
We could pass `sess` to `__wt_verbose_dump_txn_one` rather than passing in 2 arguments, both derived from `sess`?

Reply:
e391e70